source/_docs/international-regions.md

```diff
@@ -117,10 +123,10 @@ Coming soon:
     - [Fill out this survey](https://www.getfeedback.com/r/hkR9uTAJ){.external} to tell us about your needs.
 
 ## Frequently Asked Questions
-### Can I use the EU region for an existing site?
-Yes, however you must migrate your existing site to a new Site that was configured for the EU region during creation ([as described above](#create-a-new-site)).
+### Can I use an International Region for an existing site?
+Yes, however you must migrate your existing site to a new Site that was configured for International Region during creation ([as described above](#create-a-new-site)).
```

Comment:
Yes, however you must migrate your existing site to a new site that was created in the new region ([as described above(#create-a-new-site)).